project.clj

```diff
@@ -4,6 +4,8 @@
   :license {:name "Eclipse Public License"
             :url "http://www.eclipse.org/legal/epl-v10.html"}
   :dependencies [[org.clojure/clojure "1.7.0"]
+                 [org.clojure/tools.reader "0.10.0"]
+                 [org.clojure/data.json "0.2.6"]
```

Comment:
Oh you know what, are you doing JSON from the Clojure side or the Cljs side? Not sure if cheshire supports CLJS, forgot to check.